### PR TITLE
Update to Sorting section and Sorting exercise

### PR DESCRIPTION
**Pull request by zjsteyn**
I updated some of the wording referring to a "1980" sheet in the spreadsheet that was no longer available in the data downloaded at the start of the lesson.

I also included a sorting exercise for consideration based on the example of sorting the weight column and a solution was also provided.

The images displays quite large, I could resize them if required and the exercise is found useful.

### CODE REVIEW

File: _episodes/04-quality-control.md [hunk 1]
```diff
@@ -108,15 +108,43 @@ will group at the ends of the sorted data. Sort your data by each field, one at
...
 
 ![Figure of Sorting menu](../fig/sorting.png)
 
-- When you do this sort, do you notice anything strange?
+Let's try the following sorting exercise:
+
+> ## Exercise : 
+>
+> Download this data file based on the 2013 survey data to your computer: [survey_sorting_exercise](../data/survey_data_messy_quality_control.xlsx)
+>
```
Comment (tracykteal) [inline _episodes/04-quality-control.md]: For downloading the files, it should point to the FigShare file now. We just put them up there. https://ndownloader.figshare.com/files/2252083

**Comment by tracykteal** [comment]
Thanks, this looks good. I just added a comment about how to link to the excel file for download.

File: _episodes/04-quality-control.md (hunk 1 above)
Comment (ErinBecker) [inline _episodes/04-quality-control.md]: Thanks @zjsteyn for this great addition to the sorting lesson. If you could please update the link to the data file, I'll go ahead and merge this.

**Comment by ErinBecker** [comment]
Fixes https://github.com/datacarpentry/spreadsheet-ecology-lesson/issues/119

**Comment by tracykteal** [comment]
Thanks, nice addition.

**Pull request merged**